docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc

```diff
@@ -0,0 +1,156 @@
+
+=== New columns available on process instances, tasks, and jobs
+The following columns have been added to the corresponding perspectives:
+
+* Process instance list perspective: Last update and correlation key.
+* Task list: Last update, correlation key (of the associated process instance ID), process instance description (of the associated process instance).
+* Jobs perspective: Name of the associated process (if any), ID of the associated process instance (if any), Description of the associated process instance (if any).
+
+=== Advanced search filter
+You can now search for specific data by using the *Search* tab on the Jobs, Process Instances, Tasks Administration, and Execution Errors perspectives. For information about the new search feature, see the "Using the advanced search filter pages" section of the _{PRODUCT} {RELEASE} User Guide_.
+
+=== New Task List and Task Administration perspectives
+The former Task List perspective has been divided into the Tasks List and Task Administration perspectives. The Task List perspective is used by task operators to work with their assigned (or potentially assigned) tasks. The Task Administration perspective is used by administrators to manage tasks assigned to other users. This perspective is only available for users with admin or process-admin roles. It is similar to the Admin filter tab on the former Task List perspective.
+
+=== Improved validation in deployment descriptor editor
+
+The deployment descriptor editor in Business Central has been improved to validate data entered for the following:
+
+* Work item handlers
+* Event listeners
+* Marshalling strategies
+* Globals
+* Environment entries
+* Configuration
+
+Validation is performed automatically when a build is invoked or manually when a user clicks *Validate*.
+
+=== Error Handling
+The system now detects and stores execution errors such as exceptions while running a process instance or a job after the maximum number of retries has been exceeded. If an error occurs, an event is generated that contains the following data:
+
+--
+[cols="1,2",options="header"]
+|===
+| Data
+| Fields
+
+.3+| Process model
+| processModelID
+| processModelVersion
+| processModelName
+
+.2+| Process instance that has failed
+| processInstanceID
+| customProcessID
+
+.5+| Activity that has failed
+| activityID
+| activityName
+| activityType
+| iteration
+| status
+
+.2+| Error data
+| errorMessage: Message with the error that has occurred.
+| errorStack: Stack with error detail.
+|===
+--
+
+The process instance list includes a new column that displays the number of errors per process instance which are not marked as acknowledged. When you select the column, a window appears which will enables you to navigate to the new Error perspective.
```

Comment:
"will enables you" is definitely not a valid sequence in the English language. Besides, I am not sure window details even belong here. Suggested fix: "When you select the column, the system prompts you to navigate to the new Error perspective."

Reply:
Well, obviously 'which will enables' was a typo and should be 'which enables.. Thanks for the catch. I'm not sure your suggestion is better than what I have. Why wouldn't we say that a window appears which enables...? Why is system prompts better?

Reply:
If we keep window at least it should be "that enables". There is a section in the style guide on which vs that, I really doubt they would remove it. Basically, "which" always comes with a comma and is an explanatory/wide statement. "That" comes without a comma and is a narrow statement ("the specific window that...")

I think that "system prompts" is better simply because the point is not how the UI works, the point is that you can you can get to the Error perspective quickly. So a simpler structure ("the system prompts you to", 5 words, one verb) seems preferable to a more complicated structure ("a window appears that enables you to", 7 words, more imporatntly 2 verbs so the reader has to get through two separate actions in their head).

This is a matter of minimalism, so it's taste to a degree, but which/that is in the SG.